README.md

```diff
@@ -1,2 +1,18 @@
-# dp-frontend-homepage-controller
-Controller for handling homepage on ONS website
+dp-frontend-homepage-controller
+================
+
+### Getting started
+
+### Dependencies
+
+### Configuration
```

Comment:
Configs that already exist can go into here. Thinking the health check ones can go into the repo generation - will make a PR in a bit to add this. along with the getting started run `make debug` etc.

Reply:
👍